libs/services/src/SlippageService/SlippageService.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * BPS (Basis Points)
+ */
+export type Bps = number;
+
+export interface SlippageService {
+  getSlippageBps(quoteTokenAddress: string, baseTokenAddress: string): Bps;
+}
+
+// TODO: Find good name for the implementation, as Leandro don't like "Impl" suffix, just don't want to couple it to add Coingecko in its name (as that would couple it to the data-source, and the adding a name to specify the algorithm might complicate the name, as this will use some custom logic based on standard deviation, so for now as we plan to have just one implementation, I want to keep it simple. But happy to get ideas here)
```

Comment:
;)

Reply:
What about: `BasicSlippageService`
Or, `StandardSlippageService`, `DefaultSlippageService`, `NaiveSlippageService` or even `FixedSlippageService` or `DumbSlippageService` at it doesn't calculate anything (yet) :P

Reply:
You like it more as Prefix? 

The reason I usually add it a s sufix is to use it as a qualifier. Like, `SlippageServiceA`  and  `SlippageServiceB`  rather than `ASlippageService` and `BSlippageService`  

This way we go from the general "SlippageService" to the specific `SlippageServiceStrategyX`
Makes it easier to search for it in the EDI in my opinion. You type `SlippageService` and sorted you will see
- `SlippageServiceStrategyX`
- `SlippageServiceStrategyY`
- `SlippageServiceStrategyZ`

Regarding the actual prefix/suffix, I used `Impl` cause is short and typically there's just one implementation, and when there's more will be cause one is a `Mock` or a `Proxy` or `Redis` or some other qualifier, but most of the time I don't expect more than one implementation. 

I will call it `Main` for now as a trade off